image.go

```diff
@@ -59,6 +59,8 @@ type Image interface {
 	OS() (string, error)
 	OSVersion() (string, error)
 	Architecture() (string, error)
+	// Size returns the size of the manifest. If a manifest doesn't exist, it returns 0.
```

Comment:
```suggestion
	// ManifestSize returns the size of the manifest. If a manifest doesn't exist, it returns 0.
```

Reply:
Thank you. I renamed this function and forgot to update the comment.
I'll upload an updated PR soon.